docs/mine-hnt/validators/validator-score.mdx

```diff
@@ -1,39 +1,194 @@
+
+
+### DKG (Distributed Key Generation) Penalty
+* This penalty can occur if an election fails, and the Validator was part of the reason for the election to fail.
+* At the end of the election, all Validators vote and sign the election result.  If the election fails, Validators that a majority agree did not sign the election results are penalized.
+* This penalty is applied once per consensus.
+* The amount applied is a chain variable, currently set at `1.0`. The variable is viewable by running `miner ledger variables dkg_penalty`.
+* Learn more about [Distributed Key Generation](https://en.wikipedia.org/wiki/Distributed_key_generation)
+
+
+### Performance Penalty
+
+The details of the HoneyBadgerBFT protocol used is out of scope for this document, but this can be read more about [here](https://eprint.iacr.org/2016/199.pdf).  What's important is that there are a few different types of failures that will create performance penalties that can be applied to a Validator while in consensus:
+
+**Seen Penalties**
+* This penalty can occur if a majority of other Validators in the consensus group report your Validator as not being "seen"
+* This is equivalent to the Validator being offline, not available, or not communicating with the group.
+* This penalty is applied for each occurrence, which can happen multiple times during a consensus group.
+* The amount applied per occurrence is a chain variable, currently set at `0.33`. The variable is viewable by running `miner ledger variables election_seen_penalty`.
+
+**BBA (Byzantine Binary Agreement) Penalties**
+* This penalty can occur when the Validator is seen but fails to contribute a bundle of transactions and metadata to the block.  
+* Acquiring this penalty usually means the Validator is slow/underpowered, or poorly connected.
+* This may also occur if the time on your Validator is skewed vs the group time.  [Run `ntp`](https://en.wikipedia.org/wiki/Network_Time_Protocol)!
+* This penalty is applied for each occurrence, which can happen multiple times during a consensus group.
+* The amount applied per occurrence is a chain variable, currently set at `0.1`. The variable is viewable by running `miner ledger variables election_bba_penalty`.
+* Learn more about [Byzantine Agreements](https://en.wikipedia.org/wiki/Byzantine_fault) and their affect on [Consensus](https://en.wikipedia.org/wiki/Consensus_(computer_science))
+
+
+## Penalty Decay
+
+Penalties decay over time.  
+
+Each penalty, when assessed, also decays at the same rate.  This decay rate is a chain variable, able to be viewed by running `miner ledger variables penalty_history_limit`.  
+
+Every time a penalty is assessed, it begins to decay at `penalty * (1 - age/penalty_history_limit)`, where age is the # of blocks that have occurred since the penalty was put in place.  This means that as blocks are formed, each penalty decays over time, bringing a Validator back down to `0.0`.
```

Comment:
```suggestion
Every time a penalty is assessed, it begins to decay at `penalty * (1 - age/penalty_history_limit)`, where age is the number of blocks that have occurred since the penalty was put in place.  This means that as blocks are formed, each penalty decays over time, bringing a Validator back down to `0.0`.
```